packages/typespec-vs/src/VSExtension.cs

```diff
@@ -105,7 +105,7 @@ public sealed class LanguageClient : ILanguageClient
             }
             catch (Win32Exception e) when (e.NativeErrorCode == Win32ErrorCodes.ERROR_FILE_NOT_FOUND)
             {
-                throw new TypeSpecServerNotFoundException(info.FileName, e);
+                throw new TypeSpecServerNotFoundException(info, e);
```

Comment:
Hmm, there's something I didn't understand when we chatted. When info.FileName is just "tsp-server.cmd" the intent is to let process creation find tsp-server.cmd on PATH. Appending working directory in that case is misleading.

Reply:
tsp-server.cmd on PATH would come from `npm install -g @typespec/compiler`

Reply:
My idea is to expose to the user what exactly was not found in attempt to aid the troubleshooting, the file name alone does not provide any clue where it was looking and why. My hope is that appending the working directory would improve it.

Reply:
Looks like FileName can also be "node.exe" to be found on path with a .js as argument and again prepending working directory wouldn't make sense.

Reply:
cc @timotheeguerin

Reply:
Does building locally produce the VS extension file ready to be installed? I can install, break local config and see the outcome

Reply:
It's not really "exactly what was not found", though. Technically if there was a tsp-server.cmd in working directory it would use that, but that's not really ever expected since the working directory is the solution/workspace folder which shouldn't have tsp-server.cmd.

I think it would be more useful to see where ResolveLocalCompiler probed than to see the working directory prepended to a bare tsp-server.cmd that we use here when we can't resolve local compiler (and didn't have a configured path to use before that).

Perhaps we should build up a log in resolveTypeSpecServer of everything we checked and put that in the error or something.